Makefile

```diff
@@ -7,7 +7,7 @@ default: fmtcheck vet static build
 # test runs the test suite and vets the code
 test: testunit
 	@echo "==> Running Functional Tests"
-	cd govcd && go test -tags "functional" -timeout=200m -check.vv .
+	cd govcd && go test -tags "functional" -timeout=300m -check.vv .
```

Comment:
Please tell more why are you adding additional one hundred minutes to the timeout :)

Reply:
It is not related to this exact group PR. I did hit timeouts in general for full suite runs in slower envs that is why I increased the "default".